bundle/manifests/open-liberty.clusterserviceversion.yaml

```diff
@@ -43,56 +43,14 @@ metadata:
             "podName": "Specify_Pod_Name_Here",
             "traceSpecification": "*=info:com.ibm.ws.webcontainer*=all"
           }
-        },
```

Comment:
@arturdzm I don't believe CR samples for v1beta2 are needed in the CSV. Just wanted to confirm